pkg/controller/spinnakervalidating/gvk.go

```diff
@@ -18,7 +19,7 @@ func isSpinnakerRequest(req admission.Request) bool {
 func (v *spinnakerValidatingController) getSpinnakerService(req admission.Request) (v1alpha2.SpinnakerServiceInterface, error) {
 	if isSpinnakerRequest(req) {
 		svc := SpinnakerServiceBuilder.New()
-		if err := v.decoder.Decode(req, svc); err != nil {
+		if err := json.Unmarshal(req.AdmissionRequest.Object.Raw, svc); err != nil {
```

Comment:
Why is this needed? It assumes the raw is always json. It's mostly true but rules out protobuf.

Reply:
`Decoder` is not injected anymore in the new version of the SDK. Good point about protobuf, I'll look for a different way to get a decoder.